src/main/java/de/teamlapen/vampirism/core/ModEntities.java

```diff
@@ -90,79 +91,85 @@ static void registerCustomExtendedCreatures() {
      * Register convertibles for vanilla creatures and maybe for future vampirism creature as well
      */
     static void registerConvertibles() {
-        String base = REFERENCE.MODID + ":textures/entity/vanilla/%s_overlay.png";
+        Function<String, ResourceLocation> overlay = (String name) -> new ResourceLocation(REFERENCE.MODID, String.format("textures/entity/vanilla/%s_overlay.png", name));
         IVampirismEntityRegistry registry = VampirismAPI.entityRegistry();
-        registry.addConvertible(EntityType.COW, CowEntity.class, String.format(base, "cow"));
-        registry.addConvertible(EntityType.PIG, PigEntity.class, String.format(base, "pig"));
-        registry.addConvertible(EntityType.OCELOT, OcelotEntity.class, String.format(base, "cat"));
-        registry.addConvertible(EntityType.HORSE, HorseEntity.class, String.format(base, "horse"));
-        registry.addConvertible(EntityType.POLAR_BEAR, PolarBearEntity.class, String.format(base, "polarbear"));
-        registry.addConvertible(EntityType.RABBIT, RabbitEntity.class, String.format(base, "rabbit"));
-        registry.addConvertible(EntityType.SHEEP, SheepEntity.class, String.format(base, "sheep"), new ConvertedSheepEntity.ConvertingHandler());
+
+        registry.addConvertible(EntityType.COW, CowEntity.class, overlay.apply("cow"));
+        registry.addConvertible(EntityType.HORSE, HorseEntity.class, overlay.apply("horse"));
+        registry.addConvertible(EntityType.LLAMA, LlamaEntity.class, overlay.apply("llama"));
+        registry.addConvertible(EntityType.OCELOT, OcelotEntity.class, overlay.apply("cat"));
+        registry.addConvertible(EntityType.PANDA, PandaEntity.class, overlay.apply("panda"));
+        registry.addConvertible(EntityType.PIG, PigEntity.class, overlay.apply("pig"));
+        registry.addConvertible(EntityType.POLAR_BEAR, PolarBearEntity.class, overlay.apply("polarbear"));
+        registry.addConvertible(EntityType.RABBIT, RabbitEntity.class, overlay.apply("rabbit"));
+        registry.addConvertible(EntityType.SHEEP, SheepEntity.class, overlay.apply("sheep"), new ConvertedSheepEntity.ConvertingHandler());
         registry.addConvertible(EntityType.VILLAGER, VillagerEntity.class, null, new ConvertedVillagerEntity.ConvertingHandler());
-        registry.addConvertible(EntityType.LLAMA, LlamaEntity.class, String.format(base, "llama"));
-        registry.addConvertible(EntityType.PANDA, PandaEntity.class, String.format(base, "panda"));
     }
 
     static void registerEntities(IForgeRegistry<EntityType<?>> registry) {
         //simply register EntityType
-        registry.register(blinding_bat);
-        registry.register(ghost);
+        registry.register(advanced_hunter);
```

Comment:
A little bit inconsistent with some elements being initialized in static and others here. But ok

Reply:
intention consistency:
all forge registry items should be created in the register() Method.
the objects that must be used before forge assiges them to the Objectholder are created in the static initializer ( e.g. entitytypes for spawn eggs)
-> all attribute objectholder are short (no object creation)